src/libraries/System.IO.FileSystem/tests/File/EncryptDecrypt.cs

```diff
@@ -2,8 +2,6 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using System.Diagnostics;
```

Comment:
The `EncryptDecrypt_NotSupported` test method is failing with a different exception than the expected one. I'm investigating:

```
Expected: typeof(System.PlatformNotSupportedException)
Actual:   typeof(System.IO.FileNotFoundException): Could not find file 'C:\h\w\AB0D0973\w\A7620956\e\path'.
      ---- System.IO.FileNotFoundException : Could not find file 'C:\h\w\AB0D0973\w\A7620956\e\path'.
Stack Trace:
    at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
    at System.IO.File.Encrypt(String path)
```

```csharp
        [SkipOnTargetFramework(TargetFrameworkMonikers.Netcoreapp)]
        [Fact]
        public static void EncryptDecrypt_NotSupported()
        {
            Assert.Throws<PlatformNotSupportedException>(() => File.Encrypt("path"));
            Assert.Throws<PlatformNotSupportedException>(() => File.Decrypt("path"));
        }

```

Reply:
Seems that we are succeeding in calling the .NET Framework P/Invoke and encryption is supported by the underlying OS where the test is being executed, but since the path does not exist, we throw the exception message.

We were expecting to reach one of the lines that throw `PlatformNotSupportedException`:

https://referencesource.microsoft.com/#mscorlib/system/io/file.cs,324 // PAL + Not NT
https://referencesource.microsoft.com/#mscorlib/system/io/file.cs,338 // Not NTFS

But instead we now throw because the path does not exist.

I'll update this test and remove the `SkipOnTargetFramework` attribute. I suspect both .NET Framework and .NET Core would support executing the method and would throw `FileNotFoundException`.